configuration/init/systemd/tedge-mapper-az.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=tedge-mapper-az converts thin edge json measurements to thin-edge JSON for Azure format.
```

Comment:
Minor comment: we shouldn't mix `thin edge json` or `thin-edge JSON`. Although the official name seems to be `Thin Edge JSON` from our doc.